lib/tasks/lint.rake

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
 
-desc "lint Ruby, Sass and Javascript"
+desc "lint Ruby, Factorybot, Sass and Javascript"
 task "lint" do
   sh "govuk-lint-ruby --format clang"
+  sh "rake factorybot_lint"
```

Comment:
we probably should have a bundle exec here too, might want to see RAILS_ENV=test to avoid that call then shelling out again as per https://github.com/alphagov/content-publisher/pull/736/files#diff-8b9ad8593305e1b3717d3f0d09c96f9bR11